### ISSUE TITLE
Is EqualTo to allow “containsAll” mode for array and object

### ISSUE THREAD

**Issue by ashleyfrieze** [webcompere/model-assert#21]

**Comment by ashleyfrieze** [comment]
Partially fixed by #29

**Comment by ashleyfrieze** [comment]
Contains, rather than equals is available on array, but not object.

E.g.

```json
{ "a": "someA", "b": "someB"}

// does contain

{ "a": "someA" }
```

maybe:

```java
assertJson(actual)
   .where().at("/").objectContains()
   .isEqualTo(expected);
```
